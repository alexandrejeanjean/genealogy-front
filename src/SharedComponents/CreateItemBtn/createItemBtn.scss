.new-item-btn .new-card {
  width: 20em;
  height: 20em;
  margin: 0;
  img {
    width: 3.5em;
  }
}
