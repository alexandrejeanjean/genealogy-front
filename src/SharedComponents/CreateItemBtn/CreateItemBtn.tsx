import React from "react";
import { add } from "../../assets/imgPath";
import "./createItemBtn.scss";

type TChild = {
  handleClick: () => any;
};

const CreateItemBtn = ({ handleClick }: TChild) => {
  return (
    <div className="card-wrapper new-card">
      <button className="card" onClick={handleClick}>
        <img src={add} alt="ajouter" />
      </button>
    </div>
  );
};

export default CreateItemBtn;
