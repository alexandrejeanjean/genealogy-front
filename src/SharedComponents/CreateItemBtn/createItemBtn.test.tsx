import React from "react";
import { shallow } from "enzyme";
import CreateItemBtn from "./CreateItemBtn";

describe("CreateItemBtn", () => {
  it("should render correctly", () => {
    const _handleClick = jest.fn();
    const Component = shallow(<CreateItemBtn handleClick={_handleClick} />);
    expect(Component).toMatchSnapshot();
  });
});
