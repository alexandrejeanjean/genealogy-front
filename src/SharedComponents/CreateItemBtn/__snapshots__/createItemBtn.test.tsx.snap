// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateItemBtn should render correctly 1`] = `
<div
  className="card-wrapper new-card"
>
  <button
    className="card"
    onClick={[MockFunction]}
  >
    <img
      alt="ajouter"
      src={[Function]}
    />
  </button>
</div>
`;
